protocol/app/upgrades/v3.0.0/upgrade.go

```diff
@@ -101,6 +101,8 @@ func CreateUpgradeHandler(
 	return func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
 		ctx.Logger().Info("Running %s Upgrade...", UpgradeName)
 		InitializeModuleAccs(ctx, ak)
+
+		// TODO: Initialize ratelimit module params to desired state.
```

Comment:
@teddyding For you to add do when you get to it

Reply:
Could you reference this ticket: CORE-824, for tracking purpose